doc/examples/animation_widget/lib/main.dart

```diff
@@ -1,16 +1,35 @@
 import 'dart:async';
 
+import 'package:flame/flame.dart';
 import 'package:flame/animation.dart' as animation;
 import 'package:flame/sprite.dart';
-import 'package:flame/flame.dart';
+import 'package:flame/spritesheet.dart';
 import 'package:flame/position.dart';
+import 'package:flame/widgets/flame_animation_widget.dart';
+import 'package:flame/widgets/flame_sprite_widget.dart';
 import 'package:flutter/material.dart';
 
 Sprite _sprite;
+animation.Animation _animation;
```

Comment:
existing code, but since I wrote it I realized you can do `package:flutter/material.dart hide Animation`